src/scene/shader-lib/chunks-glsl/standard/frag/anisotropy.js

```diff
@@ -0,0 +1,27 @@
+export default /* glsl */`
+
+#ifdef LIT_GGX_SPECULAR
+    uniform float material_anisotropyIntensity;
+    uniform vec2 material_anisotropyRotation;
+#endif
+
+void getAnisotropy() {
+    dAnisotropy = 0.0;
+    dAnisotropyRotation = vec2(1.0,0.0);
+
+#ifdef LIT_GGX_SPECULAR
+    dAnisotropy = material_anisotropyIntensity;
+    dAnisotropyRotation = material_anisotropyRotation;
+#endif
+
+    #ifdef STD_ANISOTROPY_TEXTURE
+    dAnisotropy *= texture2DBias({STD_ANISOTROPY_TEXTURE_NAME}, {STD_ANISOTROPY_TEXTURE_UV}, textureBias).b;
```

Comment:
only sample the texture one time to a temporary and get the data out, instead of twice.